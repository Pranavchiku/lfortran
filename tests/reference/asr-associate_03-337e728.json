{
    "basename": "asr-associate_03-337e728",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/associate_03.f90",
    "infile_hash": "0857ba34ba2430fadd8ed229bd42009450944a6a6707d205cadd40e9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-associate_03-337e728.stdout",
    "stdout_hash": "d7932ccefc206239dfd082de70c94b12810332d9998964886ee0d0a8",
    "stderr": "asr-associate_03-337e728.stderr",
    "stderr_hash": "f0fe2e069146a96b823fdab7a1f5892f0aaf482422947bc75c9b4fa9",
    "returncode": 0
}