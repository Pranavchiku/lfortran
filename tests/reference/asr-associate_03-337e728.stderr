warning: Implied save feature
 --> tests/../integration_tests/associate_03.f90:4:5
  |
4 |     integer, pointer :: p1
  |     ^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/associate_03.f90:5:5
  |
5 |     integer, target :: t1 = 2, t2 = 1
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/associate_03.f90:5:5
  |
5 |     integer, target :: t1 = 2, t2 = 1
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/associate_03.f90:6:5
  |
6 |     integer :: i
  |     ^^^^^^^^^^^^ Assuming save attribute implicitly for variables
